packit/schema.py

```diff
@@ -302,6 +323,20 @@ class PackageConfigSchema(MM23Schema):
     def ordered_preprocess(self, data, **kwargs):
         data = self.rename_deprecated_keys(data, **kwargs)
         data = self.specfile_path_pre(data, **kwargs)
+        data = self.add_defaults_for_jobs(data, **kwargs)
+        return data
+
+    @staticmethod
+    def add_defaults_for_jobs(data, **_):
+        """
+        add all the fields (except for jobs) to every job so we can process only jobconfig in p-s
+        """
+        for job in data.get("jobs", []):
+            for k, v in data.items():
+                if k == "jobs":
+                    # overriding jobs doesn't make any sense
+                    continue
+                job.setdefault(k, v)
```

Comment:
So only those lines are about overwriting? Nice job!

Reply:
...and what about the other 300 lines? :D

Reply:
Yes, but there were only about argument forwarding.

I was searching for some real changes. (Without success..;) But we'll see in the packit-service.